manifests/params_agent.pp

```diff
@@ -224,6 +224,17 @@
       $ossec_syscheck_nodiff = '/etc/ssl/private.key'
       $ossec_syscheck_skip_nfs = 'yes'
 
+      # Audit
+      $audit_manage_rules                = false
+      $audit_buffer_bytes                = "8192"
+      $audit_backlog_wait_time           = "0"
+      $audit_rules                       = [
+        '-D',
```

Comment:
I also remove -D to not delete the already loaded audit rules.

Done in 16619a41df65abd9f1df59dfc262bd2ba7cde9a9

Thank you